client-app/core/utilities/helpers/index.ts

```diff
@@ -1,3 +1,7 @@
 export function sleep(ms: number, resolvedValue?: any): Promise<any> {
   return new Promise((resolve) => setTimeout(resolve, ms, resolvedValue));
 }
+
+export function truncate(str: string, length: number): string {
+  return str.length > length ? `${str.slice(0, length)}...` : str;
```

Comment:
Maybe it's better to use `&hellip;` instead of three dots?

Reply:
`&hellip;` will only work when HTML is output (`v-html`)